charts/workflow/values.yaml

```diff
@@ -51,6 +51,16 @@ global:
   host_port: 5555
   # Prefix for the imagepull secret created when using private registry
   secret_prefix: "private-registry"
+  # The public resolvable hostname to build your cluster with.
+  #
+  # This will be the hostname that is used to build endpoints such as "deis.$HOSTNAME"
+  hostname: ""
```

Comment:
@krancour thanks for the suggestions, I changed `hostname` to `platform_domain` and pulled it into the controller and out of global.